src/app.css

```diff
@@ -0,0 +1,205 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+#mainNav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+
+  /* background-color: grey; */
+  align-items: center;
+  align-content: center;
+}
+
+#ham {
+  width: 24px;
+  height: 24px;
+  cursor: pointer;
+}
+
+#logo {
+  width: 124px;
+  height: auto;
+  cursor: pointer;
+}
+
+#search-area {
+  display: flex;
+  justify-content: space-between;
+}
+
+#search-area form {
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+#search-area input {
+  width: 563px;
+  height: 20px;
+  padding: 10px;
+  font-size: 20px;
+}
+
+#search {
+  height: 100%;
+  width: 64px;
+
+  /* background-color: lightgrey; */
+  border: none;
+  position: absolute;
+  right: 0;
+}
+
+#search-area #mic {
+  border-radius: 50%;
+  height: 44px;
+  width: 44px;
+  border: none;
+  margin-left: 10px;
+  cursor: pointer;
+}
```

Comment:
- [ ] It will be a good idea to do your styling with classes preferably over IDs as classes present a lot more underlying logic which could make concepts like specificity and precedence complicated in your CSS. You can learn more about in the microverse HTML and CSS [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and also consider reading the [stackoverflow](https://stackoverflow.com/questions/12889362/what-is-the-difference-between-id-and-class-in-css-and-when-should-i-use-them) conversion for reference and better understanding.